app/components/filtered_back_link/tracker.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module FilteredBackLink
```

Comment:
I wonder whether this class should live in `app/lib/` similar to the page tracker. I'm conscious that this is the first time we have something living in a component folder that isn't the standard `Component::View` class (or the html, css, JS).

It's also used outside of the `FilteredBackLink` component in the `TraineesController` `save_filter` method.

Reply:
@AbigailMcP was thinking as its related to the component, keep it in one place.
Will move it to `lib` folder.